### ISSUE TITLE
PTLib can't be found during cross compile, but ptlib-config is found.

### ISSUE THREAD

**Issue by the-maldridge** [willamowius/h323plus#19]
I am in the process of packaging h323plus for Void Linux so that we can have h323 support in Yate.  Void makes extensive use of cross compiling, and it looks like something between the way void drives the build and how h323plus expects to be informed it is being cross compiled are out of sync:

```
checking for ptlib-config... /usr/bin/ptlib-config
checking for /usr/share/ptlib//make/lib.mak... configure: error: cannot check for file existence when cross compiling
```
([Full log here](https://github.com/void-linux/void-packages/runs/5252976425?check_suite_focus=true))

Given that I'm on 1.27.2 which explicitly lists improved cross support I feel like this is an issue with the way I'm running the build.  Have I missed something obvious?

**Comment by willamowius** [comment]
@Lastique can you comment on this as you have done the cross-compling with PTLib ?

**Comment by Lastique** [comment]
I did not cross-compile ptlib. Sorry, I cannot comment.

**Comment by the-maldridge** [comment]
From void's dev IRC:

```
15:02 <Chocimier> configure_args="ac_cv_file__usr_share_ptlib__make_lib_mak=yes"
15:02 <Chocimier> makes configure pass
15:02 <Arsen> beautiful
15:02 <Chocimier> and result of that check is unused anyway
15:03 <Chocimier> but then they use g++ rather than target compiler
```

Sounds like this check is ultimately a noop?

**Comment by willamowius** [comment]
I got feedback from somebody who successfully cross-compiled a while ago.

These versions should work fine. Maybe you can check the few commits we had since then what broke cross-compilation ?

openssl-1.1.1g

ptlib v2_10_9_3
commit 1a228d914f2790274604ce72ade4ad26ce7bca62

h323 v1_27_1
commit f6f4898a6ab1a53e13ec23fafc4e61b4671fb2d8

**Comment by willamowius** [comment]
It turns out its not H323Plus that changed, but newer versions of autoconf can't handle the file check when cross-compiling.

Should be fixed now with 8bda09298b64ae909f081dac6143ffa41992dea0.

**Comment by the-maldridge** [comment]
Fascinating.  I likely won't have a lot of time to play with this change until the weekend, but thank you for taking a look into this!
